Packages/com.unity.inputsystem/Documentation~/ActionBindings.md

```diff
@@ -927,6 +940,29 @@ Press(Keyboard.current.leftShiftKey);
 Press(keyboard.bKey);
 ```
 
+If an application needs even more control over when input events are consumed, it is possible to completely take over the handling of events by using the [`CallbackContext.HandleEvent`](api/UnityEngine.InputSystem.InputAction.CallbackContext.html#UnityEngine_InputSystem_InputAction_CallbackContext_HandleEvent) method. In the following example, event consumption is turned off for the `Shift+B` binding, and the performed handler only handles the input if the 'inventoryOpen' flag is true. The `B` binding then decides whether to perform some actions based on whether the input has been handled or not.
+
+```CSharp
+bAction.AddBinding("<Keyboard>/b");
+shiftbAction.AddCompositeBinding("OneModifier(handleInputEvents=false)")
+    .With("Modifier", "<Keyboard>/shift")
+    .With("Binding", "<Keyboard>/b");
+bool inventoryOpen;
+
+// Print something to the console when the actions are triggered.
+bAction.performed += ctx =>
+{
+    if(ctx.eventHandled == false)
+        Debug.Log("B action performed");
```

Comment:
So, assuming consume events it's off by my previous comment, this means that there's still a way to know if "another" event for this key was called, right?

Reply:
Yep, consuming events and handling events are two separate concepts. Handled means the input caused some composite binding to trigger but it doesn't by itself stop lower priority events from also triggering. That's why callbacks need to check the handled flag to see if someone else has already triggered for the current event.

Consuming is what stops an event from triggering lower priority bindings. It works by checking the handled flag, but also the InputSettings.shortcutKeysConsumeInput.